src/parser.js

```diff
@@ -220,6 +271,10 @@ class Parser {
         lngs.forEach((lng) => {
             this.resStore[lng] = this.resStore[lng] || {};
             this.resScan[lng] = this.resScan[lng] || {};
+            this.pluralSuffixes[lng] = getPluralSuffixes(lng, this.options.pluralSeparator);
```

Comment:
For languages that don't have pluralization support (e.g. "dev" for development purpose), I suggest that the `getPluralSuffixes()` function should return an empty array instead of false value, so it won't throw errors to stop execution.

Reply:
I agree. Want to fix it yourself?